riru/template/magisk_module/service.sh

```diff
@@ -26,3 +27,13 @@ if [[ "$(cat /sys/fs/selinux/enforce)" == "0" ]]; then
     chmod 640 /sys/fs/selinux/enforce
     chmod 440 /sys/fs/selinux/policy
 fi
+
+# Late props which must be set after boot_completed
+{
+    until [[ "$(getprop sys.boot_completed)" == "1" ]]; do
```

Comment:
Is it really necessary to wait for this? I thought service.sh runs late enough that HALs and most system services won't be affected.

Reply:
It is necessary, or stuff breaks on OnePlus devices. See issue #92

Reply:
It doesn't. I tried that first and it was still broken, unfortunately!

Reply:
Got it, thanks for confirming.